### PR TITLE
Move base config into BaseParamTable

### PR DESCRIPTION
**Pull request by cydrain**
Move pulsarCfg/rocksdbCfg/minioCfg into BaseParamTable

Issue: #12478

### CODE REVIEW

File: internal/util/paramtable/base_param.go [hunk 1]
```diff
@@ -29,90 +29,234 @@ import (
...
-	EtcdEndpoints []string
-	MetaRootPath  string
-	KvRootPath    string
-
-	// --- Embed ETCD ---
-	UseEmbedEtcd   bool
-	EtcdConfigPath string
-	EtcdDataDir    string
+	EtcdCfg    EtcdConfig
+	PulsarCfg  pulsarConfig
```
Comment (xiaofan-luan) [inline internal/util/paramtable/base_param.go:34]: PulsarConfig, so is MinioConfig and RocksMqConfig

Comment (cydrain) [reply internal/util/paramtable/base_param.go]: done

**Pull request merged**